src/views/PurchaseOrderDetail.vue

```diff
@@ -175,6 +176,12 @@ export default defineComponent({
     })
   },
   methods: {
+    getNumberOfItemsReceived(item: any){
+      return (parseInt(item.quantityAccepted) +  this.getPOItemAccepted(item.productId))/(item.quantity)
+    },
+    areItemsReceivedGreaterThanItemsOrdered(item: any){
```

Comment:
This method is not needed. We could use the above method only, when the fraction is greater than 1 we are over receiving